eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -4,8 +4,10 @@
 <UsageData>
   <IgnorePatterns>
     <!-- Allowed due to https://github.com/dotnet/source-build-reference-packages/pull/721 -->
+    <UsagePattern IdentityGlob="System.Collections.Immutable/*9.*" />
```

Comment:
Why are those excluded? From what I can see, `src/tasks/Microsoft.NET.WebAssembly.Webcil/Microsoft.NET.WebAssembly.Webcil.csproj` is part of the source-build graph and by changing the package version property, source-build can't pull forward to the latest version anymore.

cc @dotnet/source-build-internal

Reply:
> Why are those excluded?

There was a build error which led me to this solution:

```
 @azure-pipelines azure-pipelines / runtime-dev-innerloop (Build linux-x64 Release PortableSourceBuild)

.packages/microsoft.dotnet.arcade.sdk/9.0.0-beta.24076.5/tools/SourceBuild/AfterSourceBuild.proj#L80

.packages/microsoft.dotnet.arcade.sdk/9.0.0-beta.24076.5/tools/SourceBuild/AfterSourceBuild.proj(80,5): error : (NETCORE_ENGINEERING_TELEMETRY=AfterSourceBuild) 2 new pre-builts discovered! Detailed usage report can be found at /__w/1/s/artifacts/sb/prebuilt-report/baseline-comparison.xml.
See https://aka.ms/dotnet/prebuilts for guidance on what pre-builts are and how to eliminate them.
Package IDs are:
System.Collections.Immutable.9.0.0-alpha.1.24064.3
System.Reflection.Metadata.9.0.0-alpha.1.24064.3
```

Reply:
Adding an exclusion is most likely the wrong approach here. There are couple of options and I'm not sure which one is the best.

The algorithm that we use today for this in other repos is as following: If a project is a built task project, you want to downgrade the version, but only for the .NET Framework build.